code/modules/reagents/recipes.dm

```diff
@@ -588,6 +588,42 @@
 	new /obj/item/stack/material/plasma(get_turf(holder.my_atom), created_volume)
 	return
 
+/datum/chemical_reaction/goldsolidification
+	result = null
+	required_reagents = list("iron" = 5, "frostoil" = 5, "gold" = 20)
+	result_amount = 1
+
+/datum/chemical_reaction/goldsolidification/on_reaction(var/datum/reagents/holder, var/created_volume)
+	new /obj/item/stack/material/gold(get_turf(holder.my_atom), created_volume)
+	return
+
+/datum/chemical_reaction/uraniumsolidification
+	result = null
+	required_reagents = list("iron" = 5, "frostoil" = 5, "uranium" = 20)
```

Comment:
Did you check to make sure that this recipe has priority over the EMP reaction?

Reply:
Yep, as long as you correctly fill the recipe it will be fine. If you add too much uranium and iron the EMP will still go off wiping the extra uranium you have so you need to pour the frost oil in first.